platform/web/request.go

```diff
@@ -38,9 +44,13 @@ func init() {
 }
 
 // Decode reads the body of an HTTP request as JSON and fill a struct with its content. It's also in charge of validating the content of the struct based on gopkg.in/go-playground/validator.v9 validation tags.
-func Decode(r *http.Request, val interface{}) error {
+func Decode(r *http.Request, val interface{}, options DecoderOptions) error {
```

Comment:
I think implementing a `decode(r *http.Request, val interface{}, options DecoderOptions)` and making `Decode` calling it with Allow field false, would ease the upgrade for project not using Json schema.

Reply:
(and DecodeWithJsonSchema would call the private one instead of the public one)

Reply:
That is a good idea, but will we then ever use the correct one? In the end we'll allways end up with having this intermediate thing and I just think it's messy. It's also not a big change because adding `web.DecoderOptions{}` keeps the current behaviour.